src/harvester/wasser_de.rs

```diff
@@ -13,11 +13,14 @@
 //! | RICHTLINIE_IDS         | tags               |                                                              |
 //! | URL                    | resource           |                                                              |
 //! | JAHR_VEROEFFENTLICHUNG | issued             | http://purl.org/dc/terms/issued                              |
-//!
+//! | KOMMENTAR              | comment            |                                                              |
+//! | LAST_CHECKED           | last_checked       | Last time the wasser-DE harvester checked this source        |
```

Comment:
Is this really the time the/our harvester checked it or rather the last time it was checked by the Wasser-DE staff or do you known that Wasser-DE is running a harvester as well?

Reply:
I tried to exactly specify that this is the last time the harvester on the WasserDE Portal itself harvested this source, and not our umwelt.info harvester.
Reading it again, I obviously failed.
I'll fix the description

Reply:
> I tried to exactly specify that this is the last time the harvester on the WasserDE Portal itself harvested this source

So you do know that Wasser-DE does automatic harvesting itself? That "checking" is not someone looking into the document manually?

Reply:
> Reading it again, I obviously failed.

I don't really think so, because the intended meaning was the first thing I took from it. I am just surprised by the information that Wasser-DE is driven by a harvester as well.

Reply:
I'll leave it as-is for the moment.